### PR TITLE
realtime: added support for managed_irq

### PR DESCRIPTION
**Pull request by yarda**

### CODE REVIEW

File: tuned/profiles/functions/base.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def _check_args(cls, args, nargs_max, nargs_min = None):
 		if args is None or nargs_max is None:
 			return False
 		la = len(args)
-		return (nargs_max == 0 or nargs_max == la) and (nargs_min is None or nargs_min <= la)
+		return (nargs_max == 0 or nargs_max >= la) and (nargs_min is None or nargs_min <= la)
```
Comment (olysonek) [inline tuned/profiles/functions/base.py]: This breaks the expectation of other functions.
```
[olysonek@olysonek-rh: ~/git/tuned]$ cat /etc/tuned/test/tuned.conf 
[variables]
x=${f:assertion_non_equal:foo:3}
[olysonek@olysonek-rh: ~/git/tuned]$ sudo python3 ./tuned.py -p test
2020-03-23 11:43:50,086 INFO     tuned.daemon.application: dynamic tuning is globally disabled
2020-03-23 11:43:50,089 INFO     tuned.daemon.daemon: using sleep interval of 1 second(s)
2020-03-23 11:43:50,089 INFO     tuned.profiles.loader: loading profile: test
Traceback (most recent call last):
  File "./tuned.py", line 68, in <module>
    app = tuned.daemon.Application(args.profile, config)
  File "/home/olysonek/git/tuned/tuned/daemon/application.py", line 51, in __init__
    self._daemon = daemon.Daemon(unit_manager, profile_loader, profile_name, self.config, self)
  File "/home/olysonek/git/tuned/tuned/daemon/daemon.py", line 48, in __init__
    self._init_profile(profile_names)
  File "/home/olysonek/git/tuned/tuned/daemon/daemon.py", line 78, in _init_profile
    self.set_profile(profile_names, manual)
  File "/home/olysonek/git/tuned/tuned/daemon/daemon.py", line 95, in set_profile
    self._profile = self._profile_loader.load(profile_names)
  File "/home/olysonek/git/tuned/tuned/profiles/loader.py", line 61, in load
    self._variables.add_from_cfg(final_profile.units["variables"].options)
  File "/home/olysonek/git/tuned/tuned/profiles/variables.py", line 67, in add_from_cfg
    self.add_variable(item, cfg[item])
  File "/home/olysonek/git/tuned/tuned/profiles/variables.py", line 37, in add_variable
    v = self.expand(value)
  File "/home/olysonek/git/tuned/tuned/profiles/variables.py", line 79, in expand
    return self._functions.expand(s)
  File "/home/olysonek/git/tuned/tuned/profiles/functions/functions.py", line 90, in expand
    return re.sub(r'\\(\${f:.*})', r'\1', self._process(s))
  File "/home/olysonek/git/tuned/tuned/profiles/functions/functions.py", line 74, in _process
    self._process_func(si[1])
  File "/home/olysonek/git/tuned/tuned/profiles/functions/functions.py", line 58, in _process_func
    s = f.execute(sl[2:])
  File "/home/olysonek/git/tuned/tuned/profiles/functions/function_assertion_non_equal.py", line 22, in execute
    if args[1] == args[2]:
IndexError: list index out of range
```

Comment (yarda) [reply tuned/profiles/functions/base.py]: > This breaks the expectation of other functions.
> 
The expectation was probably wrong:
```
base.py:
	# checks arguments
	# nargs_max - maximal number of arguments, there mustn't be more arguments,
	#             if nargs_max is 0, number of arguments is unlimited
	# nargs_min - minimal number of arguments, if not None there must
	#             be the same number of arguments or more
```

File: tuned/profiles/functions/function_ternary.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import re
+from . import base
+
+class ternary(base.Function):
```
Comment (olysonek) [inline tuned/profiles/functions/function_ternary.py]: Personally I'd reserve the name 'ternary' for an actual ternary operator, i.e. `x ? foo : bar`. The name should make it clear that the function has the semantics of a ternary operator, but that the condition is a regex match. I'd name it `regex_search_ternary`. (Also note the change match -> search. This better reflects the python implementation.)

**Comment by yarda** [comment]
Made 4 arguments mandatory (upon discussion on IRC).

**Comment by yarda** [comment]
Thanks, merging.

**Pull request merged**